libs/mrwebrtc/test/data_channel_tests.cpp

```diff
@@ -224,21 +217,13 @@ TEST_P(DataChannelTests, Send) {
       });
   std::function<void(int32_t, int32_t)> state1_cb(
       [&](int32_t state, int32_t id) {
-        ASSERT_EQ(kId, id);
+        if (expected_id >= 0) {
```

Comment:
Why this changed?

Reply:
I am using this callback to handle both out-of-band and in-band channels, see line 260 and 304. Will add a comment to make it clearer.

Reply:
Fixed